DentOS_Framework/DentOsTestbed/src/dent_os_testbed/utils/test_utils/tgen_utils.py

```diff
@@ -110,7 +110,7 @@ async def tgen_utils_connect_to_tgen(device, dent_dev):
         dev_ip = str(int(swp[3:]) * 2)
         dev_groups[ixp] = [
             {
-                "name": "ip_ep1",
+                "name": ixp,
```

Comment:
Could you please create a separate commit for changes in the test framework? I would like to start upstreaming immediately.
Also, we need to log a missed feature.

Reply:
Will do. And I`ve created an issue for this: https://github.com/PLVision/dent-testing/issues/5